api/nakadi-event-bus-api.yaml

```diff
@@ -1269,15 +1269,18 @@ definitions:
         description: configuration settings for kafka storage. Only necessary if the storage type is 'kafka'
         type: object
         properties:
+          exhibitor_address:
+            description: the Zookeeper address
+            type: string
+          exhibitor_port:
+            description: the Zookeeper path
+            type: string
           zk_address:
```

Comment:
These are two different things in terms curator framework. Could you take a look at how ZooKeeperHolder.java creates EnsembleProvider? 
It creates different EnsembleProvider depending on what you provide for it.
In case of exhibitor it is hostname of one of the exhibitors, but in case of the zk it is the list of each zookeper. Alos zks are used as a backup connection strategy in case of wrong/unreachable exhibitor address.